docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1313,12 +1222,16 @@ The following error message codes may appear on the app-server log. Use the foll
   (%s) Listening on %s
 
   **Reason:**
-
-  TODO
+  *_type_* (HTTP or HTTPS) Listening on  *_app-server ports_*.
```

Comment:
*_type_* (HTTP or HTTPS) Listening on *_ip address_:_port_*.

Reply:
The bottom section is a bit confusing so just remove it. We document how to configure port and http/https in our real docs